### DISCUSSION TITLE
Audio Spectrum

### DISCUSSION THREAD

**Discussion by hmersch** [ph1p/ikea-led-obegraensad#83] [Ideas]
I have a ESP32-based AutoVisualtization on a tiny 1.3 OLED. 
I would love to transform that the obegraensad is used for a abstract post-modern display of the frequencies. 

Code base is currently ArduinoIDE thus I would either move the code to C++ or ... is there any way that the obegraensad' LEDs is controlable via ArduinoIDE? Any description on how to illumunate them from ArduinoIDE?

**Comment by raindancer2204** [comment]
I would also be intressed in a Spectrum Equilzer i had written an "Issue" for Feature Request.

**Reply by hmersch** [reply]
Could you send link thus I could like it? :-)

**Comment by raindancer2204** [comment]
Hi, I created it under the “Issues”. I don't know if it's the right place.
[https://github.com/ph1p/ikea-led-obegraensad/issues/117](url)
